group C lecture 5 notes (Rachel Zheng, Indraneel Patil)

rachelzh9 opened the pull request:
L5 OLC

Commit:
groupC lecture 5

kshitijgoel007 commented:
### Feedback
**Review Section**
1. Key takeaways from the previous lecture are presented correctly and concisely within one page.

**Summary Section**
1. The online linear classification problem definition is presented with a toy example (same as in the lecture). Any other examples might have been beneficial too; but the example from the lecture suffices.
2. There is inconsistency in notation. For example, in the hyperplane equation the authors use `R` to denote the real space. However, we also usually represent regret with the same symbol. Please correct the symbols throughout the document.
3. Minor: please use the LaTeX command `$\cdot$` to indicate inner product between `$w$` and `$x$` for the hyperplane definition.
4. Please refrain from using multiple screenshots from the lecture for algorithm description. For example, instead of Figure 2, please write the pseudocode using algorithm block.
5. The mistake bound derivation is as expected and presents the content correctly except the clash in notations with the hyperplane notation noted above.

**Outside Information**
None

### Notes to the TA
Submitted within deadline. No outside information and citations.

kshitijgoel007 reviewed (changes requested):
Please make the changes in notation as pointed out in the feedback.

Commit:
changes

rachelzh9 commented:
Thank you for the feedback. We made the changes:

- R to Real to space symbol
- changed inner dot product symbol
- changed to using algorithm block
